server/src/main/java/io/seata/server/Server.java

```diff
@@ -70,14 +75,22 @@ public static void start(String[] args) {
                 XID.setIpAddress(NetUtil.getLocalIp());
             }
         }
-
         NettyRemotingServer nettyRemotingServer = new NettyRemotingServer(workingThreads);
         XID.setPort(nettyRemotingServer.getListenPort());
         UUIDGenerator.init(parameterParser.getServerNode());
+        ConfigurableListableBeanFactory beanFactory =
+            ((GenericWebApplicationContext)ObjectHolder.INSTANCE
+                .getObject(OBJECT_KEY_SPRING_APPLICATION_CONTEXT)).getBeanFactory();
+        DefaultCoordinator coordinator = DefaultCoordinator.getInstance(nettyRemotingServer);
+        beanFactory.registerSingleton(NettyRemotingServer.class.getName(), nettyRemotingServer);
+        beanFactory.registerSingleton(DefaultCoordinator.class.getName(), coordinator);
+        if (coordinator instanceof ApplicationListener) {
+            ((GenericWebApplicationContext)ObjectHolder.INSTANCE.getObject(OBJECT_KEY_SPRING_APPLICATION_CONTEXT))
+                .addApplicationListener((ApplicationListener<?>)coordinator);
```

Comment:
just  for RaftCoordinator ?

Reply:
For now, yes, there may be greater use in the future

Reply:
done